src/Axle/Startup.cs

```diff
@@ -285,6 +286,7 @@ public void Configure(IApplicationBuilder app, IHostEnvironment env, IApplicatio
             {
                 endpoints.MapHub<SessionHub>(SessionHub.Name);
                 endpoints.MapControllers();
+                endpoints.AddSettingsTemplateEndpoint();

```

Comment:
Do you think it would be much more reasonable it change semantic here to something like:
```suggestion
                endpoints.MapSettingsTemplate();

```
It would conform to the existing convention for `IEndpointRouteBuilder` extension methods.